grakn-core/src/main/java/ai/grakn/concept/Attribute.java

```diff
@@ -91,6 +91,14 @@
     @Override
     Attribute attribute(Attribute attribute);
 
+    /**
+     * Removes the provided {@link Attribute} from this {@link Attribute}
+     * @param attribute the {@link Attribute} to be removed
+     * @return The {@link Attribute} itself
+     */
+    @Override
+    Attribute deleteAttribute(Attribute attribute);
```

Comment:
Let's leave them there for the moment. Can delete them more easily than add them back in.